src/main/java/org/adrian/com/spring/controlers/RandomDataController.java

```diff
@@ -0,0 +1,20 @@
+package org.adrian.com.spring.controlers;
+
+import org.adrian.com.spring.services.RandomTableService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/random")
+public class RandomDataController {
+    @Autowired
+    RandomTableService randomTableService;
+
+    @GetMapping("/getTable")
```

Comment:
jak metoda CRUD jest GET, to unikaj w nazwie endpoint get.
Cały temat projektoania endpointów jest ciekawy, polecam poczytać jak to zgrabnie nazywać